misc/nvt_qod.c

```diff
@@ -0,0 +1,80 @@
+/* Portions Copyright (C) 2009-2021 Greenbone Networks GmbH
+ * Based on work Copyright (C) 1998 - 2007 Tenable Network Security, Inc.
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+/**
+ * @file 
+ * @brief Converts a qod_type string into a QoD value.
+ *
+ */
+
+#include "nvt_qod.h"
+#include <glib.h>
+
+#undef G_LOG_DOMAIN
+/**
+ * @brief GLib logging domain.
+ */
+#define G_LOG_DOMAIN "lib  misc"
+
+static const struct {
+  qod_val val;
+  const char *type;
+} qod_types[] = {
+                 {EXPLOIT, "exploit"},
```

Comment:
Where can I look all these qod_types up?

Reply:
I was not able to find any documentation for qod_types. I created it first for [ospd-openvas](https://github.com/greenbone/ospd-openvas/blob/86d6637b91e779e1149301cc11c2888e0b3b000f/ospd_openvas/nvticache.py#L42) some time ago, based on [gvmd](https://raw.githubusercontent.com/greenbone/gvmd/master/src/manage_sql.c)